docs/configuration.md

```diff
@@ -541,3 +541,63 @@ config2 = config.replace(one=101)
 print(config.two)  # => 2
 print(config2.two) # => 102
 ```
+
+## Secrets management
+
+To define a secret for your BigFlow workflow, you need to use [Kubernetes Secrets](https://cloud.google.com/composer/docs/how-to/using/using-kubernetes-pod-operator#secret-config) 
+mechanism (Cloud Composer is a GKE cluster under the hood). Run the following commands in the [Cloud Shell](https://cloud.google.com/shell/docs/launching-cloud-shell#launching_from_the_console)
+or a terminal on your local machine:
+
+```shell script
+gcloud composer environments describe YOUR-COMPOSER-NAME --location <YOUR COMPOSER REGION> --format="value(config.gkeCluster)"
```

Comment:
I think we should add description what every command does.

Reply:
done